meshtastic/node_runtime/settings_runtime.py

```
@@ -0,0 +1,608 @@
+"""Settings request/response and admin command-family runtime owners."""
+
+import logging
+from typing import TYPE_CHECKING, Any, Callable
```

Ruff (0.15.6)

[warning] Import from `collections.abc` instead: `Callable`

Import from `collections.abc`

(UP035)